test/shared/utils.py

```diff
@@ -74,23 +74,38 @@ def expected_wheels(package_name, package_version, manylinux_versions=['manylinu
     # {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl
     # {python tag} and {abi tag} are closely related to the python interpreter used to build the wheel
     # so we'll merge them below as python_abi_tag
-    python_abi_tags = ['cp27-cp27m', 'cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38']
+    python_abi_tags = ['cp27-cp27m', 'cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38',
+                       'pp272-pypy_41', 'pp372-pypy3_72']
     if platform == 'linux':
         python_abi_tags.append('cp27-cp27mu')  # python 2.7 has 2 different ABI on manylinux
-        platform_tags = []
-        for architecture in ['x86_64', 'i686']:
-            for manylinux_version in manylinux_versions:
-                platform_tags.append('{manylinux_version}_{architecture}'.format(
-                    manylinux_version=manylinux_version, architecture=architecture
-                ))
+        cp_architectures = ['x86_64', 'i686']
+        pp_architectures = ['x86_64']
         def get_platform_tags(python_abi_tag):
-            return platform_tags
+            if python_abi_tag.startswith('pp'):
+                architectures = pp_architectures
+            else:
+                architectures = cp_architectures
+            return ['{manylinux_version}_{architecture}'.format(
+                        manylinux_version=manylinux_version, architecture=architecture
+                    )
+                    for manylinux_version in manylinux_versions
+                    for architecture in architectures]
```

Comment:
Not such a hack.
Another way to get the same results would be to build the `platform_tags` same way as before and filter-out invalid architectures in `get_platform_tags`, not sure about this... Or even build 2 `platform_tags` array, 1 for `cp`and the other one for `pp`. 
If keeping the way things are built in this PR, maybe:
```
architectures = {'pp': ['x86_64'], 'cp': ['x86_64', 'i686']}
def get_platform_tags(python_abi_tag):
    return ['{manylinux_version}_{architecture}'.format(
                manylinux_version=manylinux_version, architecture=architecture
            )
            for manylinux_version in manylinux_versions
            for architecture in architectures[python_abi_tag[:2]]]
```

If going to 1 array per python implementation:
```
architectures = {'pp': ['x86_64'], 'cp': ['x86_64', 'i686']}
platform_tags = {}
for python_implemention in architectures.keys():
    platform_tags[python_implemention] = []
    for architecture in architectures[python_implemention]:
        for manylinux_version in manylinux_versions:
            platform_tags[python_implemention].append('{manylinux_version}_{architecture}'.format(
                manylinux_version=manylinux_version, architecture=architecture
            ))
def get_platform_tags(python_abi_tag):
    return platform_tags[python_abi_tag[:2]]
```

Reply:
Hmmm, yeah, why not, actually; that's maybe slight cleaner :-)
Thanks!